Devices/kav_fcu.device.json

```diff
@@ -66,7 +66,7 @@
       "description": "1 = vertical speed dashes will be shown"
     },
     {
-      "id": 10,
+     "id": 10,
```

Comment:
a total no-no.

Reply:
This was just whitespace removal, I didn't change the data type :)